SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -398,7 +403,7 @@ extern const TArray<Worker_ComponentId> KnownEntityAuthorityComponents;
 //
 
 constexpr uint32 SPATIAL_SNAPSHOT_SCHEMA_HASH = 679237978;
-constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 1;
+constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 2;
```

Comment:
Incrementing the version, but not the hash seems like there's a problem somewhere.
@martin-improbable ?

Reply:
If this PR has changed the schema, then yes, I'd expect the hash to have changed. 

If this PR has changed the snapshot in some other way, e.g. added new skeleton entities, then we don't currently catch that automatically, so you have to manually increment this number.

Both hash and this number are OR'ed into a 64bit snapshot version number (for now).

Reply:
Shouldn't the new schema be added to `GDKSchemaFilePaths` though?

Reply:
Ahh I see - no new components have actually been added